e2e/playground.spec.ts

```diff
@@ -1,22 +1,34 @@
 import { Page, expect, test } from '@playwright/test';
-import { PLAYGROUND_TAG } from '../src/client/components/playground/playgroundTags';
-import { isAgentResponse, isServerResponse } from './zodUtils';
 import { blockGoogleTagManager } from './e2eTestUtils';
+import { TEST_IDS } from '../src/client/testIDs';
+
+const TEST_ID = TEST_IDS.playground;
 
 const getAgentResponse = async (page: Page) => {
   const agentResponse =
-    (await (await page.getByTestId(PLAYGROUND_TAG.agentResponseJSON)).textContent()) ?? 'Agent response not found';
-  return JSON.parse(agentResponse);
+    (await (await page.getByTestId(TEST_ID.agentResponseJSON)).textContent()) ?? 'Agent response not found';
+  return agentResponse;
 };
 
 const getServerResponse = async (page: Page) => {
   const serverResponse =
-    (await (await page.getByTestId(PLAYGROUND_TAG.serverResponseJSON)).textContent()) ?? 'Server response not found';
-  return JSON.parse(serverResponse);
+    (await (await page.getByTestId(TEST_ID.serverResponseJSON)).textContent()) ?? 'Server response not found';
+  return serverResponse;
```

Comment:
These nested awaits make it less readable, pls try to simplify this

Reply:
Thanks, fixed: https://github.com/fingerprintjs/fingerprintjs-pro-use-cases/commit/92cd7afa8ed5538543133f02f780c9bd90e0686e